docs/contributing/gatsby-governance-model.md

```diff
@@ -6,63 +6,262 @@ title: Gatsby's Governance Model
 
-Our objectives for this document are to provide current information about the management and maintenance of Gatsby open source software (OSS) and present a variety of resources and avenues of communication between Gatsby’s leadership and community.
+**All community members must follow the [Code of Conduct (CoC)](/contributing/code-of-conduct/).** Consequences for CoC violations are detailed in [moderation](#moderation).
 
-### This is a work in progress!
+## Get involved
 
-It’s important to note that Gatsby’s governance model is a work in progress that will continue to grow and evolve over time. When new governance-related issues arise, Gatsby’s leadership and Core Team will do their best to include the community and anyone directly involved in the issue in the decision-making process.
+All contributions are welcome, including issues, contributing code, new docs as well as updates and tweaks, blog posts, helping out people, and more. Read the [How to Contribute guide](/contributing/#how-to-contribute) to learn what the community can do for you and what you can do for the community.
 
-## How are decisions made about Gatsby OSS?
+### Gatsby User Collective
 
-### Contributing roles
+The [Gatsby User Collective](https://gatsbyuc.dev/)'s goal is to democratize the maintenance of community plugins to allow more folks to assist in their maintenance and to create higher quality plugins in the Gatsby ecosystem. Contributing to the Gatsby User Collective is a valuable contribution!
 
-The following groups are the primary categories of Gatsby OSS contributors.
+### Areas for contribution
 
-- **Outside Contributors** - The outstanding community of contributors who help make Gatsby awesome. These are people who’ve had a Gatsby pull request merged and adhere to the [Gatsby Code of Conduct](/contributing/code-of-conduct/).
-- **[Admins](https://github.com/orgs/gatsbyjs/teams/admin/members)** - The official team members of Gatsby, Inc. (sometimes referred to as the "inkteam"), including the Gatsby Core Team who maintain Gatsby OSS.
+As the core of Gatsby is a complex piece of software, it requires some onboarding before contributions can be made efficiently. We're clearly defining here in which areas we accept contributions at the moment and in which not. This is especially important for new features as they always have an inherent maintenance cost -- and making changes without the complete context around Gatsby are difficult here. _This list is not set in stone and can change when e.g. a "Core Maintainer (L3)" role is introduced._
```

Comment:
```suggestion
As the core of Gatsby is a complex piece of software, it requires some onboarding before contributions can be made efficiently. We're clearly defining here in which areas we accept contributions at the moment and in which not. This is especially important for new features as they always have an inherent maintenance cost -- and making changes without the complete context around Gatsby is difficult. _This list is not set in stone and can change when e.g. a "Core Maintainer (L3)" role is introduced._
```

Also if Team is actually L3 (I'm not sure if it is) then "Core Maintainer (L3)" should be "Core Maintainer (L4)"